src/main/resources/explorer/assets/plugin-explorer.css

```diff
@@ -40,18 +40,20 @@ li.active .file-tree {
 }
 .folder-expander octicon {
   font-size: 0.8em;
+  color: #fff
 }
 .root-expander {
   position: absolute;
-  padding: 2px 2px 0 5px;
-  right: 5px;
+  padding: 2px 0px 0 4px;
+  right: 15px;
   top: 10px;
   box-shadow: none;
   line-height: 1.5;
   text-align: center;
   background-color: #f4f4f4;
   border: 1px solid #ddd;
   border-radius: 3px;
+  outline: none;
 }
 .root-expander > .octicon {

```

Comment:
Only `width: 10px;` is enough in this block for apply the style of `btn-default` and centering the icon contents.